dynamic-simulation/dynamic-simulation-tool/src/main/java/com/powsybl/dynamicsimulation/tool/DynamicSimulationTool.java

```diff
@@ -131,6 +141,12 @@ public void run(CommandLine line, ToolRunningContext context) throws Exception {
 
         DynamicSimulation.Runner runner = DynamicSimulation.find();
 
+        MappingSupplier mappingSupplier = MappingSupplier.empty();
+        if (line.hasOption(MAPPING_FILE)) {
+            Path mappingFile = context.getFileSystem().getPath(line.getOptionValue(MAPPING_FILE));
+            mappingSupplier = createMappingSupplier(mappingFile, runner.getName());
+        }
```

Comment:
This option is required so this test is useless.

Reply:
Done